2022-07-07 18:11:06,425 | INFO | Check successful
2022-07-07 18:11:06,481 | ERROR | The following unhandled exception occured :
Traceback (most recent call last):
  File "C:\Users\tojak\anaconda3\envs\labeq\lib\site-packages\exopy\tasks\tasks\base_tasks.py", line 1148, in perform
    child.perform_()
  File "C:\Users\tojak\anaconda3\envs\labeq\lib\site-packages\exopy\tasks\tasks\decorators.py", line 86, in decorator
    return function_to_decorate(*args, **kwargs)
  File "C:\Users\tojak\anaconda3\envs\labeq\lib\site-packages\exopy\tasks\tasks\decorators.py", line 307, in wrapper
    return perform(obj, *args, **kwargs)
  File "c:\users\tojak\source\repos\labeq-exopy\src\labeq_exopy\tasks\tasks\instr\yokogawaGS200_tasks.py", line 43, in perform
    value = self.driver.set_ramp(value,funcVal)
AttributeError: 'YokogawaGS200' object has no attribute 'set_ramp'
2022-07-07 18:11:08,497 | INFO | Process shuting down
